openassessment/xblock/data_conversion.py

```diff
@@ -4,6 +4,17 @@
 """
 import json
 
+from django.utils.functional import Promise
+from django.utils.encoding import force_text
+from django.core.serializers.json import DjangoJSONEncoder
+
+
+class LazyEncoder(DjangoJSONEncoder):
```

Comment:
@diegomillan It would be nice to add a docstring to this class that describes its purpose.

[L128-L130](https://github.com/edx-olive-oc/edx-ora2/blob/bc2e3882c104cd35a201c72acb19471af6862dfc/openassessment/xblock/data_conversion.py#L128-L130) below seem to provide at least some of the info that would be good to include in the docstring.